src/jit/gentree.h

```diff
@@ -4635,10 +4635,22 @@ struct GenTreePutArgStk : public GenTreeUnOp
     }
 #endif // FEATURE_FASTTAILCALL
 
+#if !defined(UNIX_X86_ABI)
     unsigned getArgOffset()
     {
         return gtSlotNum * TARGET_POINTER_SIZE;
     }
+#else
+    unsigned getArgOffset()
+    {
+        return (gtSlotNum & STACK_ALIGN_STKOFFSET) * TARGET_POINTER_SIZE;
+    }
+
+    unsigned getArgPadding()
+    {
+        return gtSlotNum >> STACK_ALIGN_PADDING;
+    }
+#endif
```

Comment:
Also needed is setArgPadding
These can get/set the value of a new field gtPadAlign